src/libsync/clientsideencryption.cpp

```diff
@@ -1112,9 +1176,20 @@ void ClientSideEncryption::generateCSR(const AccountPtr &account, EVP_PKEY *keyP
         if (retCode == 200) {
             QString cert = json.object().value("ocs").toObject().value("data").toObject().value("public-key").toString();
             _certificate = QSslCertificate(cert.toLocal8Bit(), QSsl::Pem);
+            qCDebug(lcCse()) << cert;
```

Comment:
@mgallien can you add some searchable string here, so we can later map log message to a source code line? Like `cert: `?